src/main/java/edu/kis/powp/jobs2d/command/FigureComplexCommand.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command;
+
+public interface FigureComplexCommand {
+    ComplexCommand createComplexCommand();
```

Comment:
Why is it important to return ComplexCommand, not a DriverCommand. (Maybe ICompoundCommand would give you an idea.)